Tests/Snapper_test.py

```diff
@@ -2,6 +2,11 @@
 from Snapper.Snapper import Snapper
 
 
+class TestManager:
+    def on_data(self, snapshot):
+        self.snapshot = snapshot
```

Comment:
Yes, you're right. I incorrectly assumed the manager was being tested here because the assert was on a property of the manager. My bad. But yes, a MagicMack and assert_called_with is probably the way to go.